MANIFEST.in

```diff
@@ -17,6 +17,7 @@ exclude appveyor.yml
 
 recursive-include pip/_vendor *.pem
 recursive-include docs Makefile *.rst *.py *.bat
+recursive-include .tox_helpers *.py
```

Comment:
The rational for adding them was to still be able to run the testsuite with tox from a sdist, but I see that tox.ini is not included either, so I'll exclude those.